Automattic/wp-calypso
# Replace old plan icons with the new ones

#### Changes proposed in this Pull Request

* Introduces a new set of plan icons that we want to start using as a result of eCommerce and Blogger plan merge (p9jf6J-14t-p2)

Important note - this PR just replaces the icons, so the change is going to be live for everyone immediately after merging this. There's no distinction between the test group and other users.

#### Testing instructions

* Go to /plan and plans page in the signup flow, and confirm you can see the following icons and colors for corresponding tabs:

<img width="1064" alt="zrzut ekranu 2018-11-23 o 14 21 57" src="https://user-images.githubusercontent.com/205419/48945588-2d73dc00-ef2b-11e8-9a35-870720e054ee.png">
<img width="1061" alt="zrzut ekranu 2018-11-23 o 14 22 01" src="https://user-images.githubusercontent.com/205419/48945589-2d73dc00-ef2b-11e8-8606-7fa7d26d86ec.png">

* Do the same for Jetpack, confirm we still use old icons and colors there:

<img width="1055" alt="zrzut ekranu 2018-11-21 o 14 48 39" src="https://user-images.githubusercontent.com/205419/48845716-83b31480-ed9d-11e8-978f-79a074d8d66c.png">
